packages/app-admin-ui/client/pages/Home/index.js

```diff
@@ -82,8 +81,23 @@ class HomePage extends Component {
 const ListProvider = ({ getListByKey, listKeys, ...props }) => {
   // TODO: A permission query to limit which lists are visible
   const lists = listKeys.map(key => getListByKey(key));
-  const query = gqlCountQueries(lists);
 
+  if (lists.length === 0) {
+    return (
+      <Fragment>
+        <DocTitle>Home</DocTitle>
+        <PageError>
+          <p>
+            No lists defined.{' '}
+            <Link href="https://v5.keystonejs.com/guides/add-lists">Get started</Link> by creating
+            your first list.
```

Comment:
Why is `Get started` linked here? Either the entire sentence should be linked (preferred), or if only part, then the descriptive second half `by creating your first list`.